utils/nctl/sh/scenarios/common/itst.sh

```diff
@@ -208,3 +208,18 @@ function assert_node_proposed() {
     log "node-$NODE_ID created a block!"
     log "$OUTPUT"
 }
+
+# Checks logs for nodes 1-10 for equivocators
+function assert_no_equivocators_logs() {
+    local LOGS
+    log_step "Looking for equivocators in logs..."
+    for i in {1..10}; do
+        # true is because grep exits 1 on no match
+        LOGS=$(cat "$NCTL"/assets/net-1/nodes/node-"$i"/logs/stdout.log 2>/dev/null | grep -w 'Equivocated') || true
```

Comment:
Isn't this case-sensitive? [The log message is lowercase](https://github.com/casper-network/casper-node/blob/c195ed3d5de81e68a3cefdb8caf8e97fe0fa9ea4/node/src/components/consensus/era_supervisor.rs#L1142):
```
validator equivocated
```

Reply:
spoke to Andreas offline, was looking at a different message which is more periodic. Updating to his suggestion.